src/test/kotlin/coverosR3z/timerecording/EnterTimeAPITests.kt

```diff
@@ -232,31 +232,6 @@ class EnterTimeAPITests {
         assertEquals("Must be able to parse aaa as double", ex.message)
     }
 
-    /**
-     * What should happen if we send too many inputs to the API?
-     * It should complain.  We want precision.
-     *
-     * In this test we expect project, time, detail, and date to
-     * be sent.  If we get project, time, detail, FOO, and date,
-     * we throw the exception
-     *
-     * See [InexactInputsException]
-     */
-    @Test
-    fun testDoPOST_TooManyInputs() {
```

Comment:
Now we don't care about exactness with the data coming into an API, this test is thus obsolete